build/shell.sh

```diff
@@ -26,6 +26,19 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/build/common.sh"
 source "${KUBE_ROOT}/build/lib/release.sh"
 
+KUBE_RUN_COPY_OUTPUT="${KUBE_RUN_COPY_OUTPUT:-n}"
+
 kube::build::verify_prereqs
 kube::build::build_image
+
+if [[ ${KUBE_RUN_COPY_OUTPUT} =~ ^[yY]$ ]]; then
+  kube::log::status "Output from this container will be rsynced out upon completion"
+else
+  kube::log::status "Output from this container will NOT be rsynced out upon completion"
+fi
+
 kube::build::run_build_command bash || true
```

Comment:
Can't this whole script be replaced with `"${KUBE_ROOT}/build/run.sh" bash "$@"` ?

Reply:
It could, yes. However (IMO) having this is a very user-friendly way of getting into the build system.. clear entry point.

Reply:
sure, so leave the script and replace the entire body of it with that one-liner.  Otherwise it is a lot of duplicated code and error messages.